app/services/api_test_generator.py

```
@@ -0,0 +1,358 @@
+        """
+        self.together_api_key = together_api_key or settings.TOGETHER_API_KEY
+        self.together_client = Together(api_key=self.together_api_key)
+        self.parser = OpenAPIParser()
+        self.generator = HybridLocustGenerator(ai_client=self.together_client)
+
+    async def generate_tests_from_swagger_url(
+            self,
+            swagger_url: str,
+            output_dir: Path,
+            custom_requirement: Optional[str] = "",
+            host: Optional[str] = "0.0.0.0",
```

Ruff (0.13.1)

Possible binding to all interfaces

(S104)